src/multicloud-connector/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
```

Comment:
Global comments applicable to all the CLI commands
1. Register the required RPs (Microsoft.AwsConnector, Microsoft.HybridConnectivity, Microsoft.HybridCompute). 
2. Examples in the help message have some random characters. Please it to some meaningful names.